src/components/Widgets/Clock.vue

```diff
@@ -37,10 +37,14 @@ export default {
       return this.timeZone.split('/')[1].replaceAll('_', ' ');
     },
     showSeconds() {
-      return !this.options.hideSeconds;
+      if (this.options.hideSeconds) return !this.options.hideSeconds;
+      // this is the default
+      return true;
     },
     use12Hour() {
-      return !this.options.use12Hour;
+      if (this.options.use12Hour) return this.options.use12Hour;
```

Comment:
If the user set `this.options.use12Hour: false` (because they wanted 24-hr), then the first statement would skip, and would user the systems current time. I think to continue with this approach, we'd need to explicitly check for a bool (e.g. `typeof this.options.use12Hour == "boolean"`)